ui/docs/manual/design/apis/hawk/authorized-scopes.md

```diff
@@ -34,5 +36,5 @@ support for this functionality.
 certificate used temporary credentials. Just include both fields in the JSON
 object before base64 encoding into the `ext` property.
 
-**Note** the way Hawk works, the `ext` property is covered by the HMAC
-signature. So it's not possible to modify this property on-the-fly.
\ No newline at end of file
+<Warning> The way Hawk works, the `ext` property is covered by the HMAC
```

Comment:
Is there a reason for adding an empty space before "The" and after "on-the-fly."? Same thing with the other warning component.

Reply:
This was accidental. 
Removed.